### PR TITLE
release v0.9.22: changelog rpc changes

### PR DESCRIPTION
**Pull request by niklasad1**

### CODE REVIEW

File: scripts/ci/changelog/templates/_free_notes.md.tera [hunk 1]
```diff
@@ -8,3 +8,14 @@
 <!-- Such a comment will be rendered but remain invisible in the rendered markdown -->
 <!-- Edit below this line -->
 <!-- Edit above this line -->
+
+This release changes JSON-RPC server library which imposes the following changes:
```
Comment (chevdor) [inline scripts/ci/changelog/templates/_free_notes.md.tera:12]:
```suggestion
This release switches to a new JSON-RPC server library, which imposes the following changes:
```

File: scripts/ci/changelog/templates/_free_notes.md.tera [hunk 2]
```diff
@@ -8,3 +8,14 @@
...
+    - IPC transport support has removed.
```
Comment (chevdor) [inline scripts/ci/changelog/templates/_free_notes.md.tera:13]:
```suggestion
    - IPC transport support has been removed.
```

File: scripts/ci/changelog/templates/_free_notes.md.tera [hunk 3]
```diff
@@ -8,3 +8,14 @@
...
+    - DEPRECATE CLI `--rpc-max-payload`: has been deprecated in favor of `--rpc-max-request-size` or `rpc-max-response-size`.
+    - DEPRECATE CLI `--ws-max-out-buffer-capacity`: has been been deprecated in favour of `rpc-max-response-size`.
+    - DEPRECATE CLI `--ipc-path`: has been been deprecated and does nothing.
```
Comment (chevdor) [inline scripts/ci/changelog/templates/_free_notes.md.tera:16]:
```suggestion
    - DEPRECATED CLI `--rpc-max-payload`: has been deprecated in favor of `--rpc-max-request-size` or `rpc-max-response-size`.
    - DEPRECATED CLI `--ws-max-out-buffer-capacity`: has been been deprecated in favour of `rpc-max-response-size`.
    - DEPRECATED CLI `--ipc-path`: has been been deprecated and does nothing.
```

**Review by chevdor — CHANGES_REQUESTED** [review]
Few details

**Review by coderobe — APPROVED** [review]
+1 once chevdor's review has been resolved

**Review by chevdor — APPROVED** [review]

**Pull request merged**